schema/data/deploy-data.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -e
+
+database=${PGDATABASE:-cif}
+user=${PGUSER:-$(whoami)}
+host=${PGHOST:-localhost}
+port=${PGPORT:-5432}
+
+# =============================================================================
+# Usage:
+# -----------------------------------------------------------------------------
+usage() {
+    cat << EOF
+$0 [-h]
+
+Options
+
+  -h, --help
+    Prints this message
+
+EOF
+}
+
+if [ "$#" -gt 0 ]; then
+    echo "Passed $# parameters. Expected 0."
+    usage
+    exit 1
+fi
+
+__dirname="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+pushd "$__dirname" > /dev/null
+
+_psql() {
+  psql -d "$database" -h "$host" -p "$port" -U "$user" -qtA --set ON_ERROR_STOP=1 "$@" 2>&1
+}
+
+deployProdData() {
+  _psql -f "./prod/001_form.sql"
```

Comment:
Should we make this run all the files in `/prod/` ? Or do we prefer constraining which files go in there?

Reply:
I have seen this in ciip too but in there we are constraining which files to go. I'm not sure how many more insert SQL files we might need to add here but if it gets too much we might want to run all files within `/prod/` in a proper way. anyway, it's just my thoughts, what do you prefer?!

Reply:
We can leave this as is, and refactor when needed